app/models/comment.py

```diff
@@ -11,9 +11,10 @@ class Comment(db.Model):
     song = db.relationship("Song", back_populates="comments")
 
     def to_dict(self):
+        user = User.query.get(self.user_id)
```

Comment:
line 14 and 17 are needed to prevent having to do another request on the frontend.